website/content/en/highlights/ 2021-11-12-event-trottle-transform.md

```diff
@@ -0,0 +1,47 @@
+---
+date: "2021-11-16"
+title: "Event trottle transform released"
+description: "A guide that addresses the new event throttle transform"
+authors: ["barieom"]
```

Comment:
@jszwedko Have we added @barieom on the resources side so this works correctly on the site?

Reply:
@spencergilbert We hadn't yet but I fixed that in #10032

Reply:
Thanks!